typescript/sdk/src/timelock/evm/EvmTimelockDeployer.hardhat-test.ts

```diff
@@ -0,0 +1,170 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers.js';
+import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+import { keccak256 } from 'ethers/lib/utils.js';
+import hre from 'hardhat';
+
+import { TimelockController__factory } from '@hyperlane-xyz/core';
+
+import { TestChainName } from '../../consts/testChains.js';
+import { MultiProvider } from '../../providers/MultiProvider.js';
+import { randomAddress } from '../../test/testUtils.js';
+import { TimelockConfig } from '../types.js';
+
+import { EvmTimelockDeployer } from './EvmTimelockDeployer.js';
+
+chai.use(chaiAsPromised);
+
+describe('EvmTimelockDeployer', async () => {
+  let multiProvider: MultiProvider;
+  let deployer: EvmTimelockDeployer;
+  let signer: SignerWithAddress;
+  let config: TimelockConfig;
+  const PROPOSER_ROLE: string = keccak256(Buffer.from('PROPOSER_ROLE'));
+  const EXECUTOR_ROLE: string = keccak256(Buffer.from('EXECUTOR_ROLE'));
+  const CANCELLER_ROLE: string = keccak256(Buffer.from('CANCELLER_ROLE'));
```

Comment:
is it worth having these as timelock consts somewhere, as i imagine we'll be reusing this in a few places

Reply:
good call